Mathlib/Algebra/CharP/ExpChar.lean

```diff
@@ -46,6 +46,32 @@ class inductive ExpChar (R : Type u) [Semiring R] : ℕ → Prop
 #align exp_char ExpChar
 #align exp_char.prime ExpChar.prime
 
+/-- The exponential characteristic is unique. -/
+theorem ExpChar.eq {p q : ℕ} (hp : ExpChar R p) (hq : ExpChar R q) : p = q := by
```

Comment:
Is `R` explicit here? I think it should be.

Reply:
You mean "should not", right?

Reply:
Yes, sorry.